src/virtac/__main__.py

```diff
@@ -27,22 +27,25 @@ def parse_arguments():
         help="The ring mode to be used, e.g., IO4 or DIAD",
     )
     parser.add_argument(
-        "-d",
+        "-e",
         "--disable-emittance",
         help="Disable the simulator's time-consuming emittance calculation",
         action="store_true",
+        default=False,
     )
     parser.add_argument(
         "-t",
-        "--enable-tfb",
-        help="Simulate extra dummy hardware to be used by the Tune Feedback system",
+        "--disable-tfb",
+        help="Disable extra simulated hardware required by the Tune Feedback system",
         action="store_true",
+        default=False,
     )
     parser.add_argument(
         "-v",
         "--verbose",
+        type=int,
+        default=0,
         help="Increase logging verbosity",
-        action="store_true",
```

Comment:
I think we should do `action="count"` so that we support the common usage format for verbosity cli args:
```
-v
-vv
-vvv
```